docs/architecture/porting-existing-aspnet-apps/deployment-scenarios.md

```diff
@@ -9,15 +9,65 @@ ms.date: 11/13/2020
+
+In the example described above, assume the API endpoints are designated as the first part of the app to be ported to ASP.NET Core. In this case, a new ASP.NET Core app is created and hosted in IIS as a separate web *application* within the existing ASP.NET MVC web *site*. Since it will be added as a child of the original web site and will be named `api`, its default route should no longer begin with `api/` (keeping this would result in it matching URLs of the form `/api/api/endpoint`).
+
+Figure 5-1 shows how the ASP.NET Core 2.1 `api` application appears in IIS Manager as a part of the existing `DotNetMvcApp` site.
+
+![IIS Manager showing api application within .NET Framework site](./media/Figure5-1.png)
+
+**Figure 5-1**. .NET Framework Site with .NET Core application in IIS.
+
+The `DotNetMvcApp` site is hosted as an MVC 5 app running on .NET Framework version 4.7.2. It has its own IIS application pool configured in integrated mode and running .NET CLR Version v4.0.30319. The `api` application is an ASP.NET Core app running on .NET Framework version 4.6.1 (`net461`). It was added to the `DotNetMvcApp` as a new IIS application and configured to use its own Application Pool. Its Application Pool is also running in integrated mode but is configured with a .NET CLR version of `No Managed Code` since it will be executed using the [ASP.NET Core Module](https://docs.microsoft.com/aspnet/core/host-and-deploy/aspnet-core-module?view=aspnetcore-2.1). The version of the ASP.NET Core app is just an example; it could also be configured to run on .NET Core 3.1 or .NET 5, though at that point it would no longer be able to target .NET Framework libraries (see [Choose the Right .NET Core Version](choose-net-core-version.md))
+
+Configured in this manner, the only change that must be made in order for the ASP.NET Core app's APIs to be routed properly is to change its default route template from `[Route("[api/controller]")]` to `[Route("[controller]")]`.
+
+Alternately the ASP.NET Core app can be another top level web site in IIS. In this case, you can configure the original site to use a rewrite rule (with [URL Rewrite](https://www.iis.net/downloads/microsoft/url-rewrite)) that will redirect to the other app if the path starts with `/api`. The ASP.NET Core app can use a different host header for its route so that it doesn't conflict with the main app but can still respond to requests using root-based routes.
```

Comment:
```suggestion
Alternately the ASP.NET Core app can be another top-level web site in IIS. In this case, you can configure the original site to use a rewrite rule (with [URL Rewrite](https://www.iis.net/downloads/microsoft/url-rewrite)) that will redirect to the other app if the path starts with `/api`. The ASP.NET Core app can use a different host header for its route so that it doesn't conflict with the main app but can still respond to requests using root-based routes.
```